content/docs/studio/user-guide/troubleshooting.md

```diff
@@ -13,25 +13,35 @@ If you need further help, you can send us a message using `Help` on the
-- [Project does not display live metrics and plots](#project-does-not-display-live-metrics-and-plots)
-- [Project does not display DVC experiments](#project-does-not-display-dvc-experiments)
-- [Error: `dvc.lock` validation failed](#error-dvclock-validation-failed)
-- [Project does not reflect updates in the Git repository ](#project-does-not-reflect-updates-in-the-git-repository)
+- [Troubleshooting](#troubleshooting)
+  - [Support](#support)
+  - [Projects and experiments](#projects-and-experiments)
+  - [Model registry](#model-registry)
+  - [Billing and payment](#billing-and-payment)
+  - [Errors accessing your Git repository](#errors-accessing-your-git-repository)
+  - [Errors related to parsing the repository](#errors-related-to-parsing-the-repository)
+  - [Errors related to DVC remotes and credentials](#errors-related-to-dvc-remotes-and-credentials)
+    - [Errors related to DVC remotes behind firewall](#errors-related-to-dvc-remotes-behind-firewall)
+  - [Error: No data found to visualize](#error-no-data-found-to-visualize)
+  - [Error: No DVC repo was found at the root](#error-no-dvc-repo-was-found-at-the-root)
+  - [Error: Non-DVC sub-directory of a monorepo](#error-non-dvc-sub-directory-of-a-monorepo)
+  - [Error: No commits were found for the sub-directory](#error-no-commits-were-found-for-the-sub-directory)
+  - [Project got created, but does not contain any data](#project-got-created-but-does-not-contain-any-data)
+  - [Project does not contain the columns that I want](#project-does-not-contain-the-columns-that-i-want)
+    - [Show/hide columns](#showhide-columns)
+  - [Project does not contain some of my commits or branches](#project-does-not-contain-some-of-my-commits-or-branches)
+  - [Error: Missing metric or plot file(s)](#error-missing-metric-or-plot-files)
+  - [Error: Skipped big remote file(s)](#error-skipped-big-remote-files)
+  - [Error: Base commit not found](#error-base-commit-not-found)
+  - [Error: Failed to push experiment to repository](#error-failed-to-push-experiment-to-repository)
+  - [Project does not display live metrics and plots](#project-does-not-display-live-metrics-and-plots)
+  - [Project does not display DVC experiments](#project-does-not-display-dvc-experiments)
+  - [Error: `dvc.lock` validation failed](#error-dvclock-validation-failed)
+  - [Project does not reflect updates in the Git repository](#project-does-not-reflect-updates-in-the-git-repository)
+  - [I cannot find my desired Git repository in the form to add a model](#i-cannot-find-my-desired-git-repository-in-the-form-to-add-a-model)
+  - [Model registry does not display the models in my Git repositories](#model-registry-does-not-display-the-models-in-my-git-repositories)
+  - [My models have disappeared even though I did not remove (deprecate) them](#my-models-have-disappeared-even-though-i-did-not-remove-deprecate-them)
+  - [Questions or problems with billing and payment](#questions-or-problems-with-billing-and-payment)
```

Comment:
Same as before - Is there a particular reason to make this change? I feel like it was better before. If you still want to go ahead with this change, can we make it into a separate PR and limit the current PR to `force import` changes?

Reply:
Sorry @tapadipti - my IDE seems to have included these changes without me noticing.